pkgs/top-level/coq-packages.nix

```diff
@@ -23,6 +23,7 @@ let
       coq-bits = callPackage ../development/coq-modules/coq-bits {};
       coq-elpi = callPackage ../development/coq-modules/coq-elpi {};
       coq-ext-lib = callPackage ../development/coq-modules/coq-ext-lib {};
+      coq-itree = callPackage ../development/coq-modules/coq-itree { };
```

Comment:
Please use the name of the library (what you’ll write between `From` and `Require`) for the attribute and directory.

Reply:
Thanks for the review!   I've pushed new patches.